AUTHORS

```diff
@@ -3139,5 +3139,188 @@ Shigma <33423008+Shigma@users.noreply.github.com>
 atian25@qq.com <atian25@qq.com>
 Amila Welihinda <amilajack@gmail.com>
 schamberg97 <50446906+schamberg97@users.noreply.github.com>
+DrunkenPoney <mrquebec13@gmail.com>
+Christoph Tavan <dev@tavan.de>
+Clark Kozak <clarkkozak@protonmail.com>
+Michael Auderer <mike@silverstone.io>
+Linn Dahlgren <admin@opposite.biz>
+Shelley Vohr <codebytere@gmail.com>
```

Comment:
@codebytere Looks like you've used more than one address for your commits over time so I imagine we'll want to put one in .mailmap. Do you have a preference for which one is listed in AUTHORS?

Reply:
I eliminated this one and kept the other one but if you prefer the other way around, it's a simple change to `.mailmap`.